src/renderer/MMM-awesome-alexa.ts

```diff
@@ -34,7 +35,6 @@ Module.register("MMM-awesome-alexa", {
         this.sendSocketNotification("CONFIG", this.config);
         this.canvas = this.createCanvas();
         this.visualizer = new CircleVisualizer(this.canvas);
-        this.visualizer.init();
     },
 
     getDom(): HTMLElement {
```

Comment:
I think we also should not append canvas to the wrapper when visualization is disabled. So not doing this `alexaWrapper.appendChild(this.canvas);`

Reply:
I was thinking of doing it but decided against it because there is no performance hit, just an empty DOM element.

Could add it in

Reply:
An idea I just had is perhaps a `DOMBuilder` class, which takes in the config, and structures the DOM accordingly

Reply:
I think this is not necessary :P I would add just a function if anything. Because class in this case would be kinda overkill 💀